test/tests/mongo-basics/run.sh

```diff
@@ -55,35 +55,24 @@ if [[ "$testName" == *tls* ]]; then
 			openssl verify -CAfile /certs/ca.crt /certs/cert.crt
 
 		FROM $image
-		# gotta be :0 because percona's mongo doesn't have a mongodb group and estesp slayed tianon with https://github.com/moby/moby/pull/34263/files#diff-f157a3a45b3e5d85aadff73bff1f5a7cR170-R171
+		# gotta be :0 because percona's mongo doesn't have a mongodb group and estesp slayed tianon with https://github.com/moby/moby/commit/a1183dda578f531ef65766611f9e16a0636e3a17#diff-2d1cd0cbc407f38960e628655d0f29f3bf49219da7be0d1f60d2ba42a8b10bfcR170-R171
 		COPY --from=certs --chown=mongodb:0 /certs /certs
 		RUN cat /certs/cert.crt /certs/private.key > /certs/both.pem # yeah, what
 	EOD
 	image="$tlsImage"
 	mongodRunArgs+=(
 		--hostname mongo
 	)
-	# test for 4.2+ (where "s/ssl/tls/" was applied to all related options/flags)
-	# see https://docs.mongodb.com/manual/tutorial/configure-ssl/#procedures-using-net-ssl-settings
-	if docker run --rm "$image" mongod --help 2>&1 | grep -q -- ' --tlsMode '; then
```

Comment:
👀 do we still need the check for `percona:psmdb-4.2`

Reply:
```console
$ ./test/run.sh -t mongo-tls-basics percona:psmdb-6.0.6 percona:psmdb-5.0.18 percona:psmdb-4.4.22 percona:psmdb-4.2.24
testing percona:psmdb-6.0.6
        'mongo-tls-basics' [1/1]...passed
testing percona:psmdb-5.0.18
        'mongo-tls-basics' [1/1]....passed
testing percona:psmdb-4.4.22
        'mongo-tls-basics' [1/1]....passed
testing percona:psmdb-4.2.24
        'mongo-tls-basics' [1/1]....passed
```

:thinking: looks like we're fine?

Reply:
Oh, because 4.2+ switched, so we are fine. :smile: